analyses/cell-type-scimilarity/scripts/01-assign-ontology-ids.R

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env Rscript
+
+# This script is used to match cell types present in the SCimilarity model to cell ontology IDs
+# The output is a TSV file that contains `scimilarity_celltype_ontology`, `scimilarity_celltype_annotation`, and `human_readable_value`
```

Comment:
Might want to indicate that this exports 2 tsv files

Reply:
It doesn't export 2 TSV files, it only exports the one TSV file with the assigned ontology IDs. The other file I made manually and saved to the repo.

Reply:
There are definitely two tsvs in the `make_options` which is why I commented (full disclosure I didn't scroll past there 😬), so I guess the `missing` one is stale code and should be removed from the script